src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Object getRandomBall() {
```

Comment:
```suggestion
    public Ball getRandomBall() {
```
you should create an instance of Lottery in Main to use its method